### ISSUE TITLE
Feature: Support "ago" in AdjustTime and the systemd keywords "now", "today", "yesterday", "tomorrow"

### ISSUE THREAD

**Issue by gnufred** [trstringer/go-systemd-time#4]
If you look a the https://www.man7.org/linux/man-pages/man7/systemd.time.7.html, those all should be in there.

**Comment by gnufred** [comment]
I just realized that we're missing the package is missing the Timezone parsing capabilities:

```
When parsing, systemd will accept a similar syntax, but expects
       no timezone specification, unless it is given as the literal
       string "UTC" (for the UTC timezone), or is specified to be the
       locally configured timezone, or the timezone name in the IANA
       timezone database format
```

I guess we can add it in another patch. I'll create another issue if I get it done.

**Comment by gnufred** [comment]
I'm actually setting a proper Github fork to send you a proper MR. So I guess you can wait on that :P
